python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_isotropic.py

```diff
@@ -0,0 +1,170 @@
+"""
+Kriging with an isotropic covariance function
+=============================================
+
```

Comment:
I would end the docstring here.

Reply:
Why?

Reply:
No real reason except that I wrote the other examples this way when converting noteboks to scripts. Anyway, forget about this remark, variety is the soul of pleasure.